resource/file/owner/owner_test.go

```diff
@@ -0,0 +1,354 @@
+// Copyright © 2016 Asteris, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package owner_test
+
+import (
+	"context"
+	"os/user"
+	"testing"
+
+	"github.com/asteris-llc/converge/helpers/fakerenderer"
+	"github.com/asteris-llc/converge/resource"
+	"github.com/asteris-llc/converge/resource/file/owner"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestInterface ensures that owner.Owner implements the resource.Task interface
+func TestInterface(t *testing.T) {
+	t.Parallel()
+	assert.Implements(t, (*resource.Task)(nil), new(owner.Owner))
+}
+
```

Comment:
maybe add some tests to check/apply when destination is and is not a directory with recursive = true